<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayAddressUse</class>
 <widget class="QDialog" name="DisplayAddressUse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1">
   <item>
    <widget class="QTableWidget" name="m_table"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>显示类别</string>
     </property>
     <widget class="QRadioButton" name="m_radio1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>25</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>显示全部画面</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="m_radio2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>显示指定画面</string>
      </property>
     </widget>
     <widget class="QComboBox" name="m_sheetcmb">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
